components/style/servo_config_facade.rs

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /* Ref: https://github.com/servo/servo/issues/15460 for more info for why it's needed. */
+
+// TODO: write docs for this mod
+use servo_config;
```

Comment:
You've changed the `extern crate servo_config;` to `#[cfg(feature = "servo")] extern crate servo_config;` in components/style/lib.rs. 
In gecko build it tries to find this crate but fails. You can either add a cfg to this use statement or you can simply add related use statements inside servo functions.(as Manishearth wrote in issue example.)